test/lib/RuleManager.test.ts

```diff
@@ -194,6 +198,7 @@ describe('RuleManager', () => {
 					for (const res of parsedRes) {
 						expect(res.violations[0], `Message is ${res.violations[0].message}`).to.have.property("ruleName").that.is.not.null;
 					}
+					Sinon.assert.callCount(telemetrySpy, 1);
```

Comment:
Is there a way to also verify that it has set the correct engines to true? Also, do you think we are getting more value by verifying that telemetry was invoked in each test?

Reply:
That's a good idea. I'll add that.
Also, it's probably not strictly necessary to have the verification on all tests, but there's probably no harm in it. I'll try to narrow it down to 1 or 2.